cfme/cloud/provider/openstack.py

```diff
@@ -96,6 +96,10 @@ def from_config(cls, prov_config, prov_key):
             else:
                 endpoints[EventsEndpoint.name] = EventsEndpoint(**event_endpoint_config)
 
+        rsa_endpoint_config = prov_config['endpoints'].get(SSHEndpoint.name, {})
+        if rsa_endpoint_config:
+            endpoints[SSHEndpoint.name] = EventsEndpoint(**rsa_endpoint_config)
```

Comment:
This should be SSHEndpoint instance no? (typo?)

Reply:
Huge typo! :roll_eyes: